thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -15,7 +16,8 @@ export default Ember.Route.extend({
       invisibleUrns: new Set(),
       filteredUrns: new Set(),
       hoverUrns: new Set(),
-      context: { urns, anomalyRange, baselineRange, analysisRange }
+      context: { urns, anomalyRange, baselineRange, analysisRange },
+      config
```

Comment:
for now, please "declare" it as a class-member in controller.js - I'll refactor that shortly and don't want to miss anything

Reply:
Done!